### ISSUE TITLE
topic hierarchy needed for aircraft reports (AMDAR)

### ISSUE THREAD

**Issue by amilan17** [wmo-im/wis2-topic-hierarchy#182]
# Overview of proposal*
As part of the EUMETSAT GTS to WIS2 migration for incoming data, there is a need to understand what topic the following aircraft reports will be available from. 

GTS with bulletin headers like this:
- IUAD01 BABJ
- IUAD01 VHHH
- IUAD01 EGRR

These are single level aircraft reports in BUFR (we used to talk of E-AMDARs). 

# Amendment details*
add topic `weather/aviation/amdar` with description `Aircraft Meteorological Data Relay`

# Requestor(s)*
EUMETSAT

# Consultations

# Describe WIS2 data publication plans associated with the proposal

# Comments

**Comment by amilan17** [comment]
@SimonElliottEUM

**Comment by SimonElliottEUM** [comment]
@amilan17 Thanks for raising this Anna. We need to access these data as part of our routine verification of atmospheric motion vectors produced from satellite data. Once we stop ingesting data form the GTS, we will need to be able to access them with a suitable WIS 2.0 subscription

**Comment by amilan17** [comment]
sent email to Jitsuko to recruit experts to help with this.

**Comment by tomkralidis** [comment]
2024-11-26
- no updates atm

**Comment by shirley-xuelei** [comment]
Regarding the topic hierarchy of AMDAR data, I sugggest to add "amdar" under "aviation", that is, origin/a/wis2/_centre-id_/data/core/weather/aviation/amdar.  What do you think?

**Comment by SimonElliottEUM** [comment]
The solution proposed by @shirley-xuelei looks good to me.

**Comment by amilan17** [comment]
PR #231

**Comment by amilan17** [comment]
recap of discussion with Nicolas Rivaben (WMO):
 
amdar, airep, mode-s etc are sent from the aviation service to NMHS and he also recommends using a different topic. 

ET-ABO will have a meeting in September (https://community.wmo.int/en/meetings/global-aviation-stakeholders-meeting-abo-and-expert-team-aircraft-based-observations-et-abo-meeting) where this can be discussed and they can propose topic(s) as appropriate. 

Therefore, this will probably be introduced in FT2026-1 instead.

**Comment by SimonElliottEUM** [comment]
FT2026-1 would work fine for me. I can keep using JMA's GTS to WIS2 gateway as long as the service is available. I am glad that these gateways are being operated - it really helps with some aspects of the migration from GTS. Thanks DWD and JMA.

**Comment by VeraKorbulakova** [comment]
> [@amilan17](https://github.com/amilan17) [@tomkralidis](https://github.com/tomkralidis) [@StuartJMatthews](https://github.com/StuartJMatthews) this is a fair point. It makes very little difference to me where the AMDARs end up in WIS 2.0 - I just need to know the topic so that I can subscribe to them. For now, I am successfully getting all the AMDAR data I need from the JMA GTS to WIS2 gateway, using topics like this: cache/a/wis2/jp-jma-gts-to-wis2/data/core/I/U/A/D/01/EGRR and adjusting A2 and CCCC as needed.

I agree with you. I'm not sure if my question is related to this topic, but there is one point in this question that confuses me: AMDAR "core" data is on DWD gateway, "recommended" data is JMA gateway. 
For example:
"data_id": "wis2/de-dwd-gts-to-wis2/data/**recommended**/I/U/A/D/01/EGRR/IUAD01EGRR180726_C_EDZW_20250918072703_89443927"
"href": https://wis2.dwd.de/recommended/gts/egrr/A_IUAD01EGRR180726_C_EDZW_20250918072703_89443927"

"data_id": "wis2/jp-jma-gts-to-wis2/data/**core**/I/U/A/D/01/EGRR/IUAD01EGRR180717/122"
"href": https://wisdev.kishou.go.jp/data/wis2/jp-jma-gts-to-wis2/data/core/I/U/A/D/01/EGRR/IUAD01180717/A_IUAD01EGRR180717_C_RJTD_20250918072032_122.bufr

**Comment by VeraKorbulakova** [comment]
Thank you, @golfvert, you're right, I was a little mistaken. Thanks for your comment. It confused me that the same data is published as with a different license.

**Comment by efucile** [comment]
AMDAR data cannot be considered aviation as they are not regulated by ICAO. They are described as surface observations in the Manual on WIGOS. Therefore the correct topic is /weather/surface-based-observations/amdar as all the surface observation data part of WIGOS.

**Comment by amilan17** [comment]
The following topics were presented at the ABO meeting with no requests to change them

- /weather/surface-based-observations/amdar
- /weather/surface-based-observations/uas

**Comment by tomkralidis** [comment]
TT-WISMD 2025-10-06
- ACTION: @amilan17 to review with Secretariat colleagues and branch/PR

**Comment by amilan17** [comment]
in FT2026-1 PR

**Comment by SimonElliottEUM** [comment]
@amilan17 
According to comments from 06/10/2025, I expected to see AMDAR data using these topics:
/weather/surface-based-observations/amdar
/weather/surface-based-observations/uas

The data are circulating using this topic:
cache/a/wis2/eu-eumetnet-amdar/data/core/weather/surface-based-observations

Why is it not:
cache/a/wis2/eu-eumetnet-amdar/data/core/weather/surface-based-observations/amdar

**Comment by amilan17** [comment]
@SimonElliottEUM They are under review and approval through fast-track, and we expect that they will become available for implementation between 10 and 20 June.
